gym_wrapper/base_gym_env.py

```diff
@@ -0,0 +1,241 @@
+import os
+from typing import List, Optional
+
+import gym
+import numpy as np
+import pygame
+import vizdoom.vizdoom as vzd
+from gym import spaces
+from vizdoom import scenarios_path
+
+
+
+CONFIGS = [
+    ["basic.cfg", 3],  # 0
+    ["deadly_corridor.cfg", 7],  # 1
+    ["defend_the_center.cfg", 3],  # 2
+    ["defend_the_line.cfg", 3],  # 3
+    ["health_gathering.cfg", 3],  # 4
+    ["my_way_home.cfg", 5],  # 5
+    ["predict_position.cfg", 3],  # 6
+    ["take_cover.cfg", 2],  # 7
+    ["deathmatch.cfg", 20],  # 8
+    ["health_gathering_supreme.cfg", 3],  # 9
+]
+
+
+class VizdoomEnv(gym.Env):
+    def __init__(self, level, **kwargs):
```

Comment:
I feel this should define all the parameters instead of just opaque `**kwargs`. There does not seem to be _too_ many of them here :)